public/js/controllers/inviteUser.js

```diff
@@ -0,0 +1,48 @@
+angular.module('mean.system')
+  .controller('InviteUserController', ['$scope', '$http', ($scope, $http) => {
+    $scope.userNotFound = false;
+    $scope.resultFound = false;
+
+
+    $scope.searchUser = () => {
+      const { username } = $scope;
+      $scope.searchResult = [];
+      if (username.length !== 0) {
+        $http({
+          method: 'GET',
+          url: `/api/search?username=${username}`
+        }).then((response) => {
+          console.log(response)
```

Comment:
why are we logging to the console?

Reply:
That was for tracking during development. The files have been edited accordingly.